packages/client/src/views/CorrectionForm/utils.ts

```diff
@@ -99,6 +100,24 @@ export function isCorrection(declaration: IDeclaration) {
   )
 }
 
+export function motherDoesNotExistAndStateIsMother(
+  declaration: IDeclaration,
+  activeState: string
+) {
+  return (
+    !Boolean(declaration.data.mother.detailsExist) && activeState === 'mother'
```

Comment:
Notice that unit tests are failing as `declaration.data.mother` is undefined in the test scenario.

@Zangetsu101 do you think it makes more sense to do `declaration.data.mother?.detailsExist` (and same for father here) OR is it better to define empty objects for mother and father in the test suite data

```
mother: {},
father: {}
```

https://github.com/opencrvs/opencrvs-core/blob/43e67ca14cc95950a0f6c41383b3fc4af9c35202/packages/client/src/views/RegisterForm/review/ReviewSection.test.tsx#L461-L470

Reply:
The conditional was something like this `"draftData && draftData.mother && !draftData.mother.detailsExist` so I say let's go with `declaration.data.mother?.detailsExist`